spring-boot-project/spring-boot-docs/src/docs/asciidoc/messaging/pulsar.adoc

```diff
@@ -0,0 +1,144 @@
+[[messaging.pulsar]]
+== Apache Pulsar Support
+https://pulsar.apache.org/[Apache Pulsar] is supported by providing auto-configuration of the {spring-pulsar-docs}[Spring for Apache Pulsar] project.
+
+Spring Boot will auto-configure and register the classic (imperative) Spring Pulsar components when `org.springframework.pulsar:spring-pulsar` is on the classpath.
+It will do the same for the reactive components when both `org.springframework.pulsar:spring-pulsar-reactive` and Project Reactor are on the classpath.
```

Comment:
Doesn't `spring-pulsar-reactive` pull in Reactor? If so, I don't think there's any need to mention Reactor here.